taworks/taform/templates/taform/application.html

```diff
@@ -109,31 +143,35 @@
 <div id="divWordWrap">
     For each course that you have held a TA position in the past three terms, please provide the course name, instructor and your responsibilities during that time: 
 </div><br>
-Experience 1:<br>
+TA Positions Held Previous Fall Term:<br>
 {{s_form.past_position_one}} <br>
 <br>
-Experience 2:<br>
+TA Positions Held Previous Winter Term:<br>
 {{s_form.past_position_two}} <br>
 <br>
-Experience 3:<br>
+TA Positions Held Previous Spring Term:<br>
 {{s_form.past_position_three}}<br>
 <br>
-<h3>Disclosure</h3>
+<h3>Confirmation</h3>
 <div id="divWordWrap">
     I fully understand that I can only submit one application per academic term, and that once the application is submitted, no changes or modifications of any sort can be done to my application.
 </div><br>
 <div id="divWordWrap">
-    In addition to the above information, an application for a teaching assistant position implies that the applicant agrees to the following conditions of employment, and gives the department permission to verify that the student meets these conditions via examination of the student's record:
+    In addition to the above information, an application for a teaching assistant position implies that the applicant agrees to the following conditions of employment, and gives the department permission to verify that the student meets these conditions via examination of the student record.
+</div>
+<div id="divWordWrap">
+For full details on the Responsibilities and Requirements of a Management Sciences Teaching Assistantship please read in full <a href=https://uwaterloo.ca/management-sciences/graduate-students/teaching-assistantships-management-sciences>teaching assistantships management sciences</a>.
 </div>
 <div id="divWordWrap">
     <ul>
-        <li>I confirm that I do not have any outstanding English Language Proficiency requirements as part of my conditions of admission for my Graduate Studies program.</li>
-        <li>I understand that as a Full-Time registered graduate students I am not permitted to be employed by the University for more than an average of (10) ten hours per week per term.</li>
-        <li>I acknowledge that I will be available from the first day of the first month of the term to the last day of the last month of the term.</li>
+        <li><input type="checkbox" id="check1" onchange="document.getElementById('check2').disabled = !this.checked;">&nbsp;I confirm that I do not have any outstanding English Language Proficiency requirements as part of my conditions of admission for my Graduate Studies program.</li><br>
```

Comment:
Can you explain the logic here with the id="check1" but the document.getElementByID('check2').disabled.